nuget.package/MendixBinaries.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Mendix.LibGit2Sharp.NativeBinaries</id>
+    <version>1.0.1-mx</version>
```

Comment:
`-mx` implies this is a "pre-release" package. This might be confusing for other teams. It's clear it's Mendix by ID, but also if you want to add some metadata, you can use `+` instead: `1.0.1+mx`.

Talking about versioning, what if we want to update the package but still keep it on LibGit2 v1.0.1? Maybe we could use versioning scheme like `1.101.0` where `1` is just a major version of the package; `101` indicates LibGit2 version `1.0.1`, and the last `0` is the build of the package itself?

Reply:
1.0.1+mx doesnt work as + is not supported by nuget it seems the +mx does not get appended to the filename correctly
I am changing the version no as discussed 1.101.1